week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,25 +20,34 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(amount) {
+  let totalAmount = amount + amount * 0.2;
```

Comment:
Good first attempt @ferhatu! How can you improve this to make it more simple? Is there a way you can rewrite this line to use one `amount` reference?